config/frontend.ssl.properties

```diff
@@ -17,7 +17,6 @@ rest.server.blob.storage.service.factory=com.github.ambry.frontend.AmbryBlobStor
 # router
 router.hostname=localhost
 router.datacenter.name=Datacenter
```

Comment:
This should stay as "Datacenter", right? Since the other one was changed from "Data-Center" to "Datacenter."